src/promdens/promdens.py

```diff
@@ -613,22 +615,27 @@ def plot_spectrum(ics: InitialConditions) -> None:
     axs[1].set_xlim(np.min(ics.traj_index) - 1, np.max(ics.traj_index) + 1)
     axs[1].set_xlabel("IC index")
     axs[1].set_ylabel(r"$|\mu|$ (a.u.)")
-    axs[1].set_title(r"Transition dipole moments")
+    axs[1].set_title("Transition dipole moments")
     axs[1].minorticks_on()
     axs[1].tick_params('both', direction='in', which='both', top=True, right=True)
 
-    axs[2].plot(ics.spectrum[0]/ics.evtoau, ics.spectrum[-1], color=colors[-1], label='Total spectrum')
-    axs[2].fill_between(ics.spectrum[0]/ics.evtoau, ics.spectrum[-1]*0, ics.spectrum[-1], color=colors[-1], alpha=0.2)
+    energy_ev = ics.spectrum[0]/ics.evtoau
+    total_cross_section = ics.spectrum[-1]
```

Comment:
NOTE: For future refactoring, it might generally be nicer if the `spectrum` attribute had a bit more structure, since currently this indexing is rather confusing.